src/olympia/lib/settings_base.py

```diff
@@ -1719,11 +1716,11 @@ def get_redis_settings(uri):
 # Static
 STATIC_ROOT = path('site-static')
 STATIC_URL = '/static/'
-JINGO_MINIFY_ROOT = path('static')
```

Comment:
Just checking, we can still edit css & js files in `/static/` and use them locally, right?

Reply:
Yes, we do. I tested this and it seemed to work nicely, tested both `site-static` and `static` still works from my perspective.